htmresearch/regions/TMRegion.py

```diff
@@ -117,22 +116,32 @@ def getSpec(cls):
         ),
         outputs=dict(
             bottomUpOut=dict(
-              description="""The primary output of the temporal memory.""",
+              description="""The default output of TMRegion. By default this
+                   outputs the active cells. You can change this dynamically
+                   using the defaultOutputType parameter.
+                   """,
```

Comment:
No problem, like I said - just a nitpick. Better do something about the rhyming, though!